src/Components/CreatePoiForm.js

```diff
@@ -0,0 +1,144 @@
+/* eslint-disable jsx-a11y/label-has-for */
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React from 'react';
+import { connect } from 'react-redux';
+import './ComponentsCSS/createPoiForm.css';
+import axios from 'axios';
+
+const mapStateToProps = state => ({
+  name: state.name,
+  keywordOne: state.keywordOne,
+  keywordTwo: state.keywordTwo,
+  geolocCoordonnees: state.geolocCoordonnees,
+  poiKeywordsDisplay: state.poiKeywordsDisplay,
+});
+
+// get date and time to fill creation_date field in database
+const poiCreationDate = new Date().toISOString().slice(0, 19).replace('T', ' ');
+
+const getFirstKeywordId = (keyword) => {
+  switch (keyword) {
+    case 'Sport & Loisirs':
+      return 1;
+    case 'Nature & Développement Durable':
+      return 2;
+    case 'Services Publics':
+      return 3;
+    case 'Vie Quotidienne':
+      return 4;
+    case 'Transports & Mobilité':
+      return 5;
+    default:
+      return keyword;
+  }
+};
```

Comment:
Not the right way to get the id of your first level points of interests. You should the ones of your db

Reply:
it's done